debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/WriteModelStrategy.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.sink;
+
+import org.bson.BsonDocument;
+
+import com.mongodb.client.model.WriteModel;
+
+import io.debezium.connector.mongodb.converters.sink.SinkDocument;
+
+public interface WriteModelStrategy {
```

Comment:
Please add JavaDoc

Reply:
This is meant for a single default implementation and the interface might eventually change. Leaving it for the future strategies to add javadoc. Is this okay?

Filed https://issues.redhat.com/browse/DBZ-7997 for it.

Reply:
I really think that we shouldn't have undocumented interfaces at all.  It simply shouldn't be a thing in our code base.

Reply:
I think short description of the intent does not hurt. You don't need to document the structure of the interface.

Reply:
done in an extra commit